pkg/operation/botanist/component/seedadmissioncontroller/seedadmissioncontroller.go

```diff
@@ -221,6 +224,7 @@ func (g *gardenerSeedAdmissionController) Deploy(ctx context.Context) error {
 								"/gardener-seed-admission-controller",
 								fmt.Sprintf("--port=%d", port),
 								fmt.Sprintf("--tls-cert-dir=%s", volumeMountPath),
+								fmt.Sprintf("--allow-invalid-extension-resources=%t", !denyInvalidExtensionResources()),
```

Comment:
The one line function `denyInvalidExtensionResources` is not really needed (sorry I know it was me who suggested it initially).
```suggestion
								fmt.Sprintf("--allow-invalid-extension-resources=%t", !gardenletfeatures.FeatureGate.Enabled(features.DenyInvalidExtensionResources)),
```